hexant.js

```diff
@@ -0,0 +1,198 @@
+'use strict';
+
+var AnimationFrame = require('animation-frame');
+var window = require('global/window');
+var document = window.document;
+
+var HexAntWorld = require('./world.js');
+var Ant = require('./ant.js');
+var Hash = require('./hash.js');
+var OddQOffset = require('./coord.js').OddQOffset;
+var HexTileTree = require('./hextiletree.js');
+
+var BatchLimit = 256;
+
+var RulesLegend = 'W=West, L=Left, A=Ahead, R=Right, E=East, F=Flip';
+var Rules = {
+    W: -2,
+    L: -1,
+    A: 0,
+    R: 1,
+    E: 2,
+    F: 3
+};
+
+function Hexant() {
+    var el = document.querySelector('#view');
```

Comment:
I'd prefer to keep dom query selector detalis in index.js, injected into such... but we can shift that going forward